proto/viam/component/arm/v1/arm.proto

```diff
@@ -43,6 +43,15 @@ service ArmService {
     };
   }
 
+  // MoveThroughJointPositions moves every joint on a robot's arm to the specified JointPositions in the order they are specified, obeying the specified velocity and acceleration limits.
+  // This will block until done or a new operation cancels this one
+  rpc MoveThroughJointPositions(MoveThroughJointPositionsRequest) returns (MoveThroughJointPositionsResponse) {
+    option (common.v1.safety_heartbeat_monitored) = true;
+    option (google.api.http) = {
+      put: "/viam/api/v1/component/arm/{name}/joint_positions"
```

Comment:
This should have come up in the scope but this is also the path we're using for the other joint positions call and might need to change?

Reply:
Oh lol yeah that won't work.

Call it `repeated_joint_positions` maybe?

Reply:
I just went with a snake case version of the method name for maximum clarity

Reply:
Makes sense, looks good